### DISCUSSION TITLE
Add option to "rotate 90" the image

### DISCUSSION THREAD

**Discussion by dudududodododedede** [bluenviron/mediamtx#1275] [Questions]
rtsp-simple-server already supports VFlip and HFlip from libcamera but would be amazing if it also support Rot90 from libcamera:

https://libcamera.org/api-html/namespacelibcamera.html#a371b6d17d531b85c035c4e889b116571ad5251a97af7b4036b4528a54405ea2ad

**Comment by aler9** [comment]
moved here #1285
